compiler/testData/diagnostics/tests/extensions/additionalReceivers/outerClass.kt

```diff
@@ -0,0 +1,13 @@
+class Outer {
+    val x: Int = 1
+}
+
+with<Outer>
+class Inner {
+    fun bar() = x
```

Comment:
Please, add cases that check availability of `Outer` receiver within the class, too